### PR TITLE
Adding Dockerfile for providing REST API via Docker container

### PR DESCRIPTION
**Pull request by IntegralProgrammer**
NeuralCR is now capable of running in a Docker container and providing a REST API.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -150,6 +150,31 @@ A web app is available for NCR trained on HPO:
 
 https://ncr.ccm.sickkids.ca/curr/
 
+## Providing a REST API with Docker
+
+Build the Docker container:
+
+```bash
+docker build -t docker-neuralcr .
```
Comment (sdumitriu) [inline README.md]:
```suggestion
docker build -t ccmsk/neuralcr .
```

File: ncrmodel.py [hunk 2]
```diff
@@ -3,7 +3,8 @@
 import random
 import json
 import pickle 
-import fastText
+#import fastText
```
Comment (sdumitriu) [inline ncrmodel.py]:
```suggestion
```

File: train.py [hunk 3]
```diff
@@ -4,7 +4,8 @@
 import os
 from onto import Ontology
 import json
-import fastText
+#import fastText
```
Comment (sdumitriu) [inline train.py]:
```suggestion
```

**Review by sdumitriu — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (IntegralProgrammer) [reply README.md]: Fixed :heavy_check_mark:

File: ncrmodel.py (hunk 2 above)
Comment (IntegralProgrammer) [reply ncrmodel.py]: Fixed :heavy_check_mark:

File: train.py (hunk 3 above)
Comment (IntegralProgrammer) [reply train.py]: Fixed :heavy_check_mark:

**Pull request closed without merging**